client/src/slices/connection/connectionSlice.ts

```diff
@@ -62,9 +67,15 @@ const connectionSlice = createSlice({
         state.invitationUrl = undefined
         state.isLoading = false
       })
+      .addCase('demo/resetState', (state) => {
+        // eslint-disable-next-line @typescript-eslint/no-unused-vars
+        const useLegacyInvitations = state.useLegacyInvitations
+        state = initialState
+        state.useLegacyInvitations = useLegacyInvitations
```

Comment:
I think this will override the useLegacyInvitations in the intital state and can lead to bugs. I think we can do it like this:

```ts
return {
   ...initialState,
   useLegacyInvitations: state.useLegacyInvitations
}
```